packages/ember/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
We should use MIT license, forgot to say this for react :/

Reply:
Yeah it was MIT but I switched it because of react, 👍